From 3831b0c87a54f0a7ba35d0ac56530f7cd9b5c273 Mon Sep 17 00:00:00 2001
From: Michael D Labriola <mdlabriola@yahoo.com>
Date: Sat, 21 Aug 2010 13:07:20 -0400
Subject: [PATCH] missing include patch from CLFS

---
 src/path.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/src/path.c b/src/path.c
index 682969d..02b7ad2 100644
--- a/src/path.c
+++ b/src/path.c
@@ -22,6 +22,7 @@
 /* Handling of path search of included files via the builtins "include"
    and "sinclude".  */
 
+#include <sys/stat.h>
 #include "m4.h"
 
 struct includes
-- 
1.7.1.1

