From 1d81c7baa6afd83d7aaeaf48f67c45912e9aca76 Mon Sep 17 00:00:00 2001
From: Michael D Labriola <mdlabriola@yahoo.com>
Date: Sun, 15 Aug 2010 21:47:23 -0400
Subject: [PATCH 1/2] disable linking to libgcc_eh

---
 Makeconfig |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makeconfig b/Makeconfig
index 9db1ee4..30d0db9 100644
--- a/Makeconfig
+++ b/Makeconfig
@@ -547,12 +547,12 @@ else
   libunwind = -lunwind
 endif
 ifneq ($(have-as-needed),yes)
- libgcc_eh := -lgcc_eh $(libunwind)
+ libgcc_eh :=  $(libunwind)
 else
  libgcc_eh := -Wl,--as-needed -lgcc_s$(libgcc_s_suffix) $(libunwind) -Wl,--no-as-needed
 endif
 gnulib := -lgcc $(libgcc_eh)
-static-gnulib := -lgcc -lgcc_eh $(libunwind)
+static-gnulib := -lgcc  $(libunwind)
 libc.so-gnulib := -lgcc
 endif
 ifeq ($(elf),yes)
-- 
1.7.1.1

